src/api/db/schema.rb

```diff
@@ -100,41 +99,41 @@
   create_table "backend_infos", id: :integer, charset: "utf8mb4", collation: "utf8mb4_unicode_ci", options: "ENGINE=InnoDB ROW_FORMAT=DYNAMIC", force: :cascade do |t|
     t.string "key", null: false, collation: "utf8mb3_unicode_ci"
     t.string "value", null: false, collation: "utf8mb3_unicode_ci"
-    t.datetime "created_at"
-    t.datetime "updated_at", precision: 6
+    t.datetime "created_at", precision: nil
+    t.datetime "updated_at"
```

Comment:
The spec is failing with this or do you just do those out of precaution?

Reply:
I have done this to follow the [Upgrading Ruby on Rails guide](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#rails-version-is-now-included-in-the-active-record-schema-dump).